TestsCalculator/Pages/CalculatorPage.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using OpenQA.Selenium;
+
+namespace TestsCalculator.Pages
+{
+    public class CalculatorPage
+    {
+        private IWebDriver _driver;
+
+        public CalculatorPage(IWebDriver driver)
+        {
+            _driver = driver;
+        }
+
+        public IWebElement AmountField => _driver.FindElement(By.Id("amount"));
+        public IWebElement PercentField => _driver.FindElement(By.Id("percent"));
+        public IWebElement TermField => _driver.FindElement(By.Id("term"));
+        public IWebElement DayDropdown => _driver.FindElement(By.Id("day"));
```

Comment:
I suggest to return the **SelectElement**, as you are using it, not the **IWebElement**.
```suggestion
        public SelectElement Day => new SelectElement(_driver.FindElement(By.Id("day")));
```

in tests:
```
        // new SelectElement(calculatorPage.DayDropdown).SelectByText(day);
        calculatorPage.Day.SelectByText(day);
```